plugins/kotlin/completion/tests/testData/basic/common/ambiguousSuperMethod.kt

```diff
@@ -0,0 +1,22 @@
+interface A {
+    fun foo(i: Int) {}
+}
+
+interface B {
+    fun foo(i: Int) {}
+}
+
+fun A.bar() {}
+fun B.bar() {}
+
+class C : A, B {
+    override fun foo(j: Int) {
+        super.fo<caret>
+    }
+}
+
+// EXIST: { lookupString:"foo", tailText:"(i: Int)" }
+// EXIST: { lookupString:"foo", tailText:"(i: Int)" }
+// EXIST: { lookupString:"foo", tailText:"(i)" }
```

Comment:
Should not here be "j" as parameter instead?

Reply:
It should. This is a bug in FE1.0.

Reply:
For the `ambiguousSuperMethod.fir.kt ` there is also `i` instead of `j`, but debug idea instance shows correct `j` here

Reply:
Ah... I missed `// FIR_COMPARISON`, which is why the test is passing...